pkgs/development/interpreters/ruby/rubygems/default.nix

```
@@ -13,6 +13,12 @@ stdenv.mkDerivation rec {
     ./0001-add-post-extract-hook.patch
     ./0002-binaries-with-env-shebang.patch
     ./0003-gem-install-default-to-user.patch
+    # Ensure tmp directory are not left behind
+    # https://github.com/rubygems/rubygems/pull/4610
+    (fetchpatch {
+      url = "https://github.com/rubygems/rubygems/pull/4610/commits/71057bb881f00cec983d3a42b07655834301f73e.patch";
```

Please use the commit url
```suggestion
      url = "https://github.com/rubygems/rubygems/commit/71057bb881f00cec983d3a42b07655834301f73e.patch";
```